Prog/AddNew/Add_new_geographies_2020.sas

```diff
@@ -0,0 +1,48 @@
+/**************************************************************************
+ Program:  Add_new_geographies_2020.sas
+ Library:  PresCat
+ Project:  Urban-Greater DC
+ Author:   Elizabeth Burton
+ Created:  7/15/2022
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Add 2020 census geographies, 2017 nhbd clusters, 2022 wards to Preservation Catalog. 
+ 
+ Modifications:
+**************************************************************************/
+
+%include "\\sas1\DCdata\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( PresCat )
+%DCData_lib( MAR )
+
+proc sort data = PresCat.Building_geocode out=Building_geocode;
+	by Bldg_address_id;
+run;
+
+proc sort data = MAR.address_points_2022_07 out=address_points_2022_07(rename=(ADDRESS_ID=bldg_address_id));
+	by ADDRESS_ID;
+run;
+
+data left_join;
+	merge Building_geocode (in=a) 
+		address_points_2022_07 (in=b);
+	by bldg_address_id;
+
+	if a then output; 
+run;
+
+  %Finalize_data_set( 
+    /** Finalize data set parameters **/
+    data=Building_geocode,
```

Comment:
@eburton9 The data= parameter is the data set that will eventually be finalized. You are using the Building_geocode which is the original file. Your updated file is called left_join, so that's the name you should put here.